variables.tf

```diff
@@ -182,3 +182,8 @@ variable "nlb_subnet_cidr_blocks" {
   type        = list(string)
 }
 
+variable "kms_key_id" {
+  description = "KMS customer managed key (CMK) ID for encrypting application logs."
+  default     = ""
```

Comment:
Should we default to requiring a KMS key, so we don't forget to enable it later?

Reply:
Yeah, I was being conservative. I'll make it required and include an example of how to set it up.

Reply:
I have required this feature in all 2.1.X and up versions. 🎉